redaxo/src/core/lib/console/db/set_connection.php

```diff
@@ -30,19 +30,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $config = rex_file::getConfig($configFile);
 
         $changed = false;
-        if ($input->hasOption('host')) {
+        if ($input->getOption('host')) {
             $config['db'][1]['host'] = $input->getOption('host');
             $changed = true;
         }
-        if ($input->hasOption('login')) {
+        if ($input->getOption('login')) {
             $config['db'][1]['login'] = $input->getOption('login');
             $changed = true;
         }
-        if ($input->hasOption('password')) {
+        if ($input->getOption('password')) {
```

Comment:
Es sollte möglich sein, ein leeres Passwort zu setzen. Vermutlich musst du dafür explizit mit `null ===` vergleichen, oder?
Würde ich bei allen hier machen. Also auch wenn jemand `--host ""` setzt, sollte das nicht ignoriert werden, sondern man sollte dann den Fehler bekommen, dass die Verbindung nicht klappt.

Reply:
stimmt, aber in diesem Falle `null !==`.
Habs angepasst und getestet